src/main/java/com/mastercard/app/petstore/PetstoreApplication.java

```diff
@@ -22,19 +22,19 @@ public class PetstoreApplication {
     EmployeeFlowExample employeeFlowExample;
 
     @Bean
-    PetFlowExample petFlowExample() throws ApiException {
```

Comment:
why are we removing the ApiException here?

Reply:
The generation of the class no longer throws an ApiException. That can happen when petUseCaseFlow() is ran.